usersession/userd/privileged_desktop_launcher_test.go

```diff
@@ -74,6 +75,11 @@ func (s *privilegedDesktopLauncherSuite) SetUpTest(c *C) {
 		filepath.Join(dirs.GlobalRootDir, "/usr/share"),
 		filepath.Dir(dirs.SnapDesktopFilesDir),
 	}, ":"))
+
+	restore := systemd.MockSystemctl(func(cmd ...string) ([]byte, error) {
+		return []byte("systemd 246\n+PAM and more"), nil
```

Comment:
Should we detect when systemctl is called with some other parameters, what would suggest missing mocking and fail in such case?

Reply:
I can do this - fwiw, it will fail naturally because the other systemctl commands expect a very different output. but happy to make this explicit :)

Reply:
Not a blocker, if you want to have a go at it, then it's fine.